gcc/rust/lang.opt

```diff
@@ -43,6 +43,10 @@ frust-dump-
 Rust Joined RejectNegative
 -frust-dump-<type>	Dump Rust frontend internal information.
 
+frust-mangling=
+Rust Joined RejectNegative
+-frust-mangling=<version> Choose which version to use for name mangling (legacy, v0)
```

Comment:
Maybe this could be a good case for TargetVariable + Enum? So that the possible version are known/handled automatically?

Reply:
Done in 37f3b02